plugin/hover.py

```diff
@@ -119,18 +122,19 @@ def run_async() -> None:
 
         sublime.set_timeout_async(run_async)
 
+    def _create_hover_request(
+        self, session: Session, point: int
+    ) -> Union[TextDocumentPositionParams, ExperimentalTextDocumentRangeParams]:
+        if session.get_capability('experimental.rangeHoverProvider'):
+            region = first_selection_region(self.view)
+            if region is not None and region.contains(point):
+                return text_document_range_params(self.view, region)
+        return text_document_position_params(self.view, point)
+
```

Comment:
I'm nit picked but it's better if the methods are ordered in the order they are used. It help when reading the code from top to bottom. So would ask you to move this one below `request_symbol_hover_async`